great_expectations/__init__.py

```diff
@@ -11,7 +11,11 @@
 
 del get_versions  # isort:skip
 
-from great_expectations.data_context import get_context
+from great_expectations.data_context import (
+    get_context,
+    get_store_from_global_context,
```

Comment:
Do we need this in the top level init?

Reply:
i don't feel strongly about this, but the other option is to leave it one level higher in `data_context`. I like that models don't even need to think about `data_context` in order to get the store they need.